src/open_inwoner/templates/pages/cases/status.html

```diff
@@ -11,55 +11,93 @@
+
+                    {% if case.internal_upload_enabled %}
+                        <h2 class="h2" id=>{% trans "Document toevoegen" %}</h2>
+                        {% if case.case_type_config_description %}
+                            <p class="p">{{ case.case_type_config_description }}</p>
+                        {% else %}
+                            <p class="p p--muted">
+                                {% blocktranslate with max_filesize=openzaak_config.max_upload_size allowed_extensions=case.allowed_file_extensions|join:', ' %}
+                                    Grootte max. {{ max_filesize }} MB, toegestane bestandsformaten {{ allowed_extensions }}.
+                                {% endblocktranslate %}
+                            </p>
+                        {% endif %}
+
+                <section id="formOpenUpload" class="form__open-upload">
+
+                    <div class="upload-interface">
+                    {#  if local upload is enabled #}
+                    {% render_form id="document-upload" form=form method="POST" submit_text=_("Bestand uploaden") enctype="multipart/form-data" show_required=True %}
+                        {% csrf_token %}
+                        <section id="infoContainer" class="info-container form__control--info">
+                            <div class="drive">
+                                <fieldset class="fieldset-container">
+                                    <div class="drive">{% icon icon="insert_drive_file" outlined=True icon_position="before" %}</div>
+                                    <div class="fieldset__content">
+                                        <div id="uploadName" class="upload-info upload-info__name"><span>{% trans "(Titel bestand)" %}</span></div>
+                                        <div id="uploadSize" class="upload-info upload-info__size"><span>{% trans "(Bestandsgrootte)" %}</span></div>
+                                    </div>
+                                    <div class="close">{% button for="document-upload" id="closeUpload" icon="cancel" type='reset' text='' transparent=True icon_outlined=True icon_position="before" %}</div>
+                                </fieldset>
+                            </div>
+                        </section>
+
+                        <div class="form__control--info">{% input form.title no_label=False no_help=True class="label input" id="id_title" %}</div>
+
+                        <div class="form__control--info">{% input form.type no_label=False no_help=True icon="expand_more" icon_position="after" class="label input" id="id_title" extra_classes="file-type__select" %}</div>
+
+                        <div class="form__control upload">
+                            <div class="upload__container">
+                                {% file_input form.file text="Bestanden uploaden" no_label=False no_help=True extra_classes="file-upload" %}
```

Comment:
Let's make the text translatable as well. The required field indicator, in the button, looks a bit weird to me, but you know better.

![required](https://user-images.githubusercontent.com/63058071/234009040-fe30d6e8-a823-4c13-9faa-d81a4f32cc71.png)